airbyte-webapp/src/components/connection/CatalogTree/next/StreamConnectionHeader.tsx

```diff
@@ -16,12 +20,25 @@ export const StreamConnectionHeader: React.FC = () => {
   } = useConnectionEditService();
   const sourceDefinition = useSourceDefinition(source.sourceDefinitionId);
   const destinationDefinition = useDestinationDefinition(destination.destinationDefinitionId);
-
+  const sourceStyles = classnames(styles.connector, styles.source);
+  const destinationStyles = classnames(styles.connector, styles.destination);
   return (
     <div className={styles.container}>
-      <div className={styles.connector}>{renderIcon(sourceDefinition.icon)} Source</div>
-      <FontAwesomeIcon icon={faArrowRight} />
-      <div className={styles.connector}>{renderIcon(destinationDefinition.icon)} Destination</div>
+      <div className={sourceStyles}>
+        {renderIcon(sourceDefinition.icon)}{" "}
```

Comment:
The icon styles predate this PR and match the values in Figma.  However, some logos appear _very_ small (Salesforce for one).  I'm in favor of adjusting it, but want to see if others agree before I go straying from the designs.

Reply:
Looking at the preview, the logos look too small for me because of the irregular size. Either the width or width is consistently 15 px or the height but not both because it will make the icon shrink to fit both. Another difference with Figma is that the icons will not render as sharply as they do, so for example the Postgres icon doesn't look great there. We may need to revisit this with design to look acceptable in the implementation.

Reply:
agree